cmd/observe/observe.go

```diff
@@ -408,12 +414,13 @@ func getFlows(client observer.ObserverClient, req *observer.GetFlowsRequest) err
 			return err
 		}
 
-		flow := getFlowResponse.GetFlow()
-		if flow == nil {
-			continue
+		if flow := getFlowResponse.GetFlow(); flow != nil {
```

Comment:
nit: might be cleaner to use type switch.

example: https://github.com/cilium/hubble/blob/cfcad093611faa9e26cbe1750998a8f5cab2b555/pkg/printer/printer.go#L167